samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Widgets.razor

```diff
@@ -171,7 +171,7 @@
     private readonly List<Bookmark> _bookmarks =
     [
         new("Angeles National Forest",
-            thumbnail: new BookmarkThumbnail("/_content/dymaptic.GeoBlazor.Core.Sample.Shared/images/Blazor-API-60px.png"),
+            thumbnail: new BookmarkThumbnail("/_content/dymaptic.GeoBlazor.Core.Sample.Shared/images/Blazor-API-60px.webp"),
```

Comment:
i have tried multiple configurations to try to get this to display.  No dice.  But, in the current samples, the png does not display here either.  BookmarkThumbnail specifically asks for a string.  In the bookmarks widget page, these thumbnails have a different shape vs a url:
![image](https://github.com/user-attachments/assets/dedf172b-62bf-407c-ab71-f35cd71c5ee7)

![image](https://github.com/user-attachments/assets/f61a8ea0-d796-486b-bc02-f42c9ae931f8)